tests/rptest/tests/pandaproxy_test.py

```diff
@@ -185,6 +193,19 @@ def _create_consumer(self, group_id, headers=HTTP_CREATE_CONSUMER_HEADERS):
                             headers=headers)
         return res
 
+    @cluster(num_nodes=3)
+    def test_get_brokers(self):
+        """
+        Test get_brokers returns the set of node_ids
+        """
+        brokers_raw = self._get_brokers()
+        brokers = brokers_raw.json()["brokers"]
+
+        nodes = enumerate(self.redpanda.nodes, 1)
```

Comment:
@BenPope why is the first node skipped?

Reply:
I think they just start at 1?

Reply:
@BenPope ahh I see what you are doing now. I'd recommend querying for the broker's IDs instead so that we don't make assumptions about how the redpanda service is assigning ids.

`assert set(brokers) == {self.redpanda.idx(n) for n in self.redpanda.nodes}`

Reply:
https://github.com/vectorizedio/redpanda/issues/2020